tests/integration/test_storage_delta/test.py

```diff
@@ -148,15 +170,88 @@ def get_delta_metadata(delta_metadata_file):
     return combined_json
 
 
-def create_delta_table(node, table_name, bucket="root", use_delta_kernel=False):
-    node.query(
-        f"""
-        DROP TABLE IF EXISTS {table_name};
-        CREATE TABLE {table_name}
-        ENGINE=DeltaLake(s3, filename = '{table_name}/', url = 'http://minio1:9001/{bucket}/')
-        SETTINGS allow_experimental_delta_kernel_rs={use_delta_kernel}"""
+def get_creation_expression(
```

Comment:
Why not leave the old `create_delta_table` and just add `storage_type` to it? This rename would make backports more difficult, therefore better to stay with the old `create_delta_table` with default `storage_type="s3"`

Reply:
sure, I can keep the same function name supporting multiple storage types & other flags